README.md

```diff
@@ -20,9 +20,8 @@ This project is used to manage IAM permissions for
   repository to create users.
 - User Services account and roles must have been created using
   [`cisagov/cool-accounts-userservices`](https://github.com/cisagov/cool-accounts-userservices).
-- Terraform in [`cisagov/cool-sharedservices-networking`](https://github.com/cisagov/cool-sharedservices-networking)
-  must have been applied.
-- Terraform in [`cisagov/cool-userservices-networking`](https://github.com/cisagov/cool-userservices-networking)
+- Terraform in
+  [`cisagov/cool-domain-manager-networking`](https://github.com/cisagov/cool-domain-manager-networking)
```

Comment:
I think this repo should really be named "cool-domainmanager-networking" but that's not a hill I willing to die on.

Reply:
I know I pretty much violated our naming convention with all of the `cool-domain-manager-*` repos I recently created, but in retrospect, if we were starting over, I would have preferred that we hyphenated all of our repo names like this.  I think the extra hyphen is worth it for improved readability.

Reply:
In this case I prefer that the triptych "cool", "domainmanager", and "networking" be made clear.  Then it's obvious at a glance what this repo is about.

We'll agree to disagree, but let each other live in peace.

Reply:
☮️ ✌️